pkg/controller/styra.go

```diff
@@ -22,22 +22,22 @@ import (
 
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/config"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/datasource"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/secret"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/stack"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/config"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/datasource"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/secret"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/stack"
 	"github.com/crossplane-contrib/provider-styra/pkg/controller/system"
 )
 
 // Setup creates all Cluster API controllers with the supplied logger and adds
 // them to the supplied manager.
 func Setup(mgr ctrl.Manager, l logging.Logger, rl workqueue.RateLimiter) error {
 	for _, setup := range []func(ctrl.Manager, logging.Logger, workqueue.RateLimiter) error{
-		config.Setup,
-		secret.SetupSecret,
+		// config.Setup,
+		// secret.SetupSecret,
 		system.SetupSystem,
-		stack.SetupStack,
-		datasource.SetupDataSource,
+		// stack.SetupStack,
+		// datasource.SetupDataSource,
```

Comment:
Are you doing all this then in `system.Setup` ?

Reply:
These changes were made by mistake.